src/AspNetCoreModuleV2/CommonLib/hostfxr_utility.cpp

```diff
@@ -182,7 +196,7 @@ HOSTFXR_UTILITY::GetHostFxrParameters(
     }
 
     // Check if the absolute path is to dotnet or not.
-    if (struAbsolutePathToDotnet.EndsWith(L"dotnet.exe") || struAbsolutePathToDotnet.EndsWith(L"dotnet"))
+    if (HOSTFXR_UTILITY::IsDotnetExecutable(struAbsolutePathToDotnet.QueryStr()))
```

Comment:
You already have STRU why turn it into cstring and back?

Reply:
Mostly for the sake of a better api for outside functions calling IsDotnetExecutable. I'll change it though as all other calls use STRU.